.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: "1.18.0-rc.1"
```

Comment:
try to update action to `actions/setup-go@v4`
https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#how-to-use